src/Native/Unix/System.Security.Cryptography.Native.Apple/pal_x509chain.c

```diff
@@ -186,7 +186,7 @@ static void MergeStatusCodes(CFTypeRef key, CFTypeRef value, void* context)
         // It doesn't represent a new error code, and we're still getting the old ones, so
         // just ignore it for now.
     }
-    else if (CFEqual(keyString, CFSTR("NonEmptySubject")))
+    else if (CFEqual(keyString, CFSTR("NonEmptySubject")) || CFEqual(keyString, CFSTR("GrayListedKey")))
```

Comment:
@danmosemsft This line we'll want to send to release/2.1 at whatever rate we say we support Mojave